README.md

```diff
@@ -1,2 +1,43 @@
 # playground-for-dotnet
 Simple examples that help you get started with Appwrite + .NET (=❤️)
+
+## Get Started
+
+This playground doesn't include any C# best practices but rather intended to show some of the most simple examples and use cases of using the Appwrite API in your .NET application.
+
+### System Requirements
+* A system with [.NET](https://dotnet.microsoft.com/) installed.
+* You have a running instance of Appwrite (localhost in most cases).
+* Create a project in Appwrite.
+* Generate an API Key in Appwrite.
+
+### Installation
+Clone this repository:
+```sh
+git clone https://github.com/appwrite/playground-for-dotnet.git
+```
+Restore all project dependencies:
+```sh
+dotnet restore
+```
+Replace `[ENDPOINT]`, `[PROJECT_ID]` and `[API_KEY]` with your credentials in *Program.cs*.
+
+To run your Playground, run following command:
+
+```sh
+dotnet run
+```
+
+## Contributing
+
+All code contributions - including those of people having commit access - must go through a pull request and approved by a core developer before being merged. This is to ensure proper review of all the code.
+
+We truly ❤️ pull requests! If you wish to help, you can learn more about how you can contribute to this project in the [contribution guide](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md).
+
+## Security
+
+For security issues, kindly email us [security@appwrite.io](mailto:security@appwrite.io) instead of posting a public issue in GitHub.
+
+## Follow Us
+
+Join our growing community around the world! Follow us on [Twitter](https://twitter.com/appwrite_io), [Facebook Page](https://www.facebook.com/appwrite.io), [Facebook Group](https://www.facebook.com/groups/appwrite.developers/) or join our live [Gitter community](https://gitter.im/appwrite/community) for more help, ideas and discussions.
```

Comment:
We should probably replace Gitter with Discord (https://appwrite.io/discord), which repo you used as template? we should probably fix it there too.

Reply:
Copy & pasted it from https://github.com/appwrite/playground-for-web 👍

Reply:
Opened https://github.com/appwrite/playground-for-web/pull/20 👍